holesky/run.sh

```diff
@@ -3,6 +3,63 @@
 
 . ./.env
 
+# Initialize variables
+NODE_ECDSA_KEY_FILE_HOST=""
+NODE_ECDSA_KEY_PASSWORD=""
+OPERATION_TYPE=""
+QUORUMS=""
+
+# Loop through command-line arguments
+# shellcheck disable=SC2039
+while [[ $# -gt 0 ]]; do
+    key="$1"
+
+    case $key in
+        --node-ecdsa-key-file-host)
+            NODE_ECDSA_KEY_FILE_HOST="$2"
+            shift
+            ;;
+        --node-ecdsa-key-password)
+            NODE_ECDSA_KEY_PASSWORD="$2"
+            shift
+            ;;
+        --operation-type)
+            OPERATION_TYPE="$2"
+            shift
+            ;;
+        --quorums)
+            QUORUMS="$2"
+            shift
+            ;;
+        *)  # Unknown flag
+            echo "Unknown option: $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+# Check if flags are provided
```

Comment:
Not that we're setting these variables from flags, it should probably support `--help/-h` to tell which flags are supported/required

Reply:
good call. will add.

Reply:
added